92-eth-discovery.md

```diff
@@ -0,0 +1,51 @@
+## Preamble
+
+    Idea: #92-disc-v5-research
+    Title: Research Ethereum discovery protocol
+    Status: Draft
+    Created: 2018-03-12
+
+## Summary
+Discovery protocol is essential for long-term scaling of the Status app. It will improve security and reliability of the app and prepare it for the beta launch. 
+
+## Swarm Participants
+- Lead Contributor: @adambabik
+- Testing & Evaluation: TBD
```

Comment:
We need one here. Who is best suited for this type of technical evaluation you think?

Reply:
That's a good question. No one has really talked to us about this idea yet.